apps/prairielearn/package.json

```diff
@@ -144,6 +144,7 @@
     "memorystream": "^0.3.1",
     "mersenne": "0.0.4",
     "mime": "^4.0.7",
+    "minimist": "^1.2.8",
```

Comment:
The previous PR also added a dependency on commander in apps/prairielearn, was that an oversight?

Reply:
Indeed! Removed in 9aedae8.